doc/output-details-and-examples/src/output-files/eplusout-perflogcsv.tex

```diff
@@ -0,0 +1,69 @@
+\section{eplusout\_perflog.csv}
+
+An output file ending with \_perflog.csv is created whenever the PerformancePrecisionTradeoffs object is present. This log file consists of echoing the settings in the PerformancePrecisionTradeoffs object, 
+some overall building energy results, the runtime of the simulation, oscillation outputs, and the number of warning and severe messages. The \_perflog.csv
+file is appended to instead of replaced each time a simulation is run to facilitate evaluation of the performance objects options and their impact on the runtime and other factors so  
+the user can determine which combination is best for their particular input file. It includes a timestamp. Since it is a CSV (comma separated value) file, it is easily opened in spreadsheet software. Please ensure that file is 
+not open in the spreadsheet while EnergyPlus is running so that the file can be appended to. If the spreadsheet program is open with the file, it will be locked and not able to be appended. The fields in the
+file are shown below.
+
+
+\begin{itemize}
+\item
+  Program
+\item
+  Version
+\item
+  TimeStamp
+\item
+  Use Coil Direct Solution
+\item
+  Zone Radiant Exchange Algorithm
+\item
+  Override Mode
+\item
+  Number of Timesteps per Hour
+\item
+  Minimum Number of Warmup Days
+\item
+  SuppressAllBeginEnvironmentResets
+\item
+  MaxZoneTempDiff
+\item
+  Electricity ABUPS Total [J]
+\item
+  Natural Gas ABUPS Total [J]
+\item
+  Additional Fuel ABUPS Total [J]
+\item
+  District Cooling ABUPS Total [J]
+\item
+  District Heating ABUPS Total [J]
+\item
+  Water ABUPS Total [m3]
+\item
+  Values Gathered Over [hours]
+\item
+  Facility Any Zone Oscillating Temperatures Time [hours]
+\item
+  Facility Any Zone Oscillating Temperatures During Occupancy Time [hours]
+\item
+  Facility Any Zone Oscillating Temperatures in Deadband Time [hours]
+\item
+  Run Time [seconds]
+\item
+  Run Time [string]
+\item
+  Number of Warnings
+\item
+  Number of Severe
+\end{itemize}
+
+When using this file it is good to set PerformancePrecisionTradeoffs to default values the first time to establish a baseline then change one input at a time and rerunning to understand the impact of that input. The total runtime decrease 
+and the impact on energy and the ocscillation variables should be considered prior to committing to a specific PerformancePrecisionTradeoffs scheme.
+
+An example is shown below:
+
+\begin{lstlisting}
+
+\end{lstlisting}
```

Comment:
Did you mean to put something in this block?

Reply:
Good question

Reply:
No need for this, I put examples in the IOref docs